tests/integration/iop/test_dag_pull_sftp.py

```diff
@@ -77,13 +77,5 @@ def test_pull_from_sftp(iop_empty_repo):
             },
             {"xml": "extracted/aca95c.xml/aca95c.xml"},
         ]
+
         assert iop_empty_repo.find_all() == expected_files
-        assert sorted(iop_empty_repo.get_all_raw_filenames()) == sorted(
```

Comment:
in some test we keep this in some other we don't. I will suggest to keep them in all ☺️